packages/components/src/DataTable/DataTable.mdx

```diff
@@ -0,0 +1,550 @@
+---
```

Comment:
I just want to say, I went through this whole page of documentation and it is really well done. So many use cases are covered which will make usage easier for many! I am really impressed! 💯